### PR TITLE
branding: add AOSC OS branding

### PR DESCRIPTION
**Pull request by eatradish**
Hi!

I am an AOSC OS developer. We made a cockpit branding for AOSC OS. AOSC OS is a Linux distribution (https://aosc.io).

### CODE REVIEW

File: src/branding/aosc/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,10 @@
+aoscbrandingdir = $(datadir)/cockpit/branding/aosc
+
+dist_aoscbranding_DATA = \
+	src/branding/aosc/branding.css \
+	$(NULL)
+
+install-data-hook::
+	ln -sTfr $(DESTDIR)/usr/share/cockpit/aosc-branding/backdrop.jpg $(DESTDIR)$(aoscbrandingdir)/backdrop.jpg
+	ln -sTfr $(DESTDIR)/usr/share/cockpit/aosc-branding/apple-touch-icon.png $(DESTDIR)$(aoscbrandingdir)/apple-touch-icon.png
+	ln -sTfr $(DESTDIR)/usr/share/cockpit/aosc-branding/favicon.ico $(DESTDIR)$(aoscbrandingdir)/favicon.ico
```
Comment (martinpitt) [inline src/branding/aosc/Makefile.am:10]: These are weird symlink targets -- these are supposed to point to some existing system resources, such as /usr/share/pixmaps/ or so -- files that are delivered by the operating system. Where would they come from?

**Review by martinpitt — CHANGES_REQUESTED** [review]
I'm really sorry that this slipped through the cracks! If you are still interested, I'll help you land this soon.

You also need to add this new Makefile.am to the top-level ./Makefile.am to the others, where it e.g. says

    include src/branding/scientific/Makefile.am

Please also add it to the removal list in tools/debian/rules.

I'm happy to do these minor cleanups for you, but I can't help you with finding and deciding for the correct logo/background images. Also, do you want to add a `#badge`?

Thanks!

File: src/branding/aosc/Makefile.am (hunk 1 above)
Comment (eatradish) [reply src/branding/aosc/Makefile.am:10]: Done

**Comment by eatradish** [comment]
> I'm really sorry that this slipped through the cracks! If you are still interested, I'll help you land this soon.
> 
> You also need to add this new Makefile.am to the top-level ./Makefile.am to the others, where it e.g. says
> 
> ```
> include src/branding/scientific/Makefile.am
> ```
> 
> Please also add it to the removal list in tools/debian/rules.
> 
> I'm happy to do these minor cleanups for you, but I can't help you with finding and deciding for the correct logo/background images. Also, do you want to add a `#badge`?
> 
> Thanks!

Done

**Comment by martinpitt** [comment]
Thanks @eatradish , but this still leaves the main issue unaddressed -- the symlinks point to a strange and most probably invalid place.

**Pull request closed without merging**